docs/associations/source/reference/level2_asn_technical.rst

```diff
@@ -0,0 +1,151 @@
+
+.. _asn-level2-example:
+   
+Example Association
+-------------------
+
+The following example will be used to explain the contents of an association::
+
+    {
+        "asn_rule": "Asn_Lv2Image",
+        "asn_pool": "jw82600_001_20160304T145416_pool",
+        "program": "82600",
+        "asn_type": "image2",
+        "products": [
+            {
+                "name": "test_lrs1_rate",
+                "members": [
+                    {
+                        "expname": "test_lrs1_rate.fits",
+                        "exptype": "SCIENCE"
+                    }
+                ]
+            },
+            {
+                "name": "test_lrs2bkg_rate",
+                "members": [
+                    {
+                        "expname": "test_lrs2bkg_rate.fits",
+                        "exptype": "SCIENCE"
+                    }
+                ]
+            },
+            {
+                "name": "test_lrs2_rate",
+                "members": [
+                    {
+                        "expname": "test_lrs2_rate.fits",
+                        "exptype": "SCIENCE"
+                    },
+                    {
+                        "expname": "test_lrs2bkg_rate.fits",
+                        "exptype": "BACKGROUND"
+                    }
+                ]
+            }
+        ]
+    }
+
+Association Meta Keywords
+-------------------------
+
+The following are the top-level, or meta, keywords of an association.
+
+program
+  Program number for which this association was created.
+  
+asn_type
+  The type of association represented. See :ref:`level3-asn-association-types`
+
+asn_id
+  The association id. The id is what appears in the :ref:`asn-DMS-naming`
+  
+asn_pool
+  Association pool from which this association was created.
+
+asn_rule
+  Name of the association rule which created this association.
+  
+version_id
+  Version identifier. DMS uses a time stamp with the format
+  `yyyymmddthhmmss`
+  Can be None or NULL
+
+constraints
+  List of constraints used by the association generator to create this
+  association. Format and contents are determined by the defining
+  rule.
+
+
+`products` Keyword
+^^^^^^^^^^^^^^^^^^
+
+Association products have to components:
+
+name
+  The string template to be used by Level 2b processing tasks to create
+  the output file names. The product name, in general, is a prefix on
+  which the individual pipeline and step modules will append whatever
+  suffix information is needed.
+
+members
+  This is a list of the exposures to be used by the Level 2b processing
+  tasks. This keyword is explained in detail in the next section.
+
+`members` Keyword
+^^^^^^^^^^^^^^^^^
+
+`members` is a list of objects, each consisting of the following
+keywords
+
+expname *required*
+  The exposure file name
+
+exptype *required*
+  Type of information represented by the exposure. Possible values are
+
+  * `SCIENCE`
+  * `BACKGROUND`
+  * `LEAKCAL`
```

Comment:
The step that subtracts the "leakcal" exposures is known as the "imprint" subtraction step and the effect has already been referred to - at least within calibration circles - as "imprint." I think the "leakcal" term is terminology from the APT/PPS folks? As far as I can tell the NIRSpec folks have always called it "imprint."